packages/client-python/src/rocketride/core/dap_client.py

```
@@ -318,19 +318,31 @@ async def connect(self, timeout: Optional[float] = None) -> None:
+            if getattr(self, '_client_display_version', None):
+                auth_args['clientVersion'] = self._client_display_version
+            request = {
+                'type': 'request',
+                'command': 'auth',
+                'seq': self._next_seq(),
+                'arguments': auth_args,
+            }
+            try:
+                response = await self.request(request, timeout=auth_timeout)
+            except Exception:
+                await self._transport.disconnect('Auth request failed', True)
```

Ruff (0.15.7)

[warning] Boolean positional value in function call

(FBT003)